src/main/flight/mixer.c

```diff
@@ -540,6 +540,17 @@ void FAST_CODE mixTable()
 
         motorValueWhenStopped = getReversibleMotorsThrottleDeadband();
         mixerThrottleCommand = constrain(rcCommand[THROTTLE], throttleRangeMin, throttleRangeMax);
+
+        #ifdef USE_DSHOT
```

Comment:
The coding convention is that `#ifdef` and other compiler directives would be intended to the left from the rest of the code.
This one change and looks goot to me

Reply:
Thanks a lot for your hints and patience! I've corrected the align.